packages/eslint-plugin/TSLINT_RULE_ALTERNATIVES.md

```diff
@@ -185,7 +185,7 @@ It lists all TSLint rules along side rules from the ESLint ecosystem that are th
 | [`one-line`]                        | 🌟  | [`brace-style`][brace-style] or [Prettier]                                          |
 | [`one-variable-per-declaration`]    | 🌟  | [`one-var`][one-var]                                                                |
 | [`ordered-imports`]                 | 🌓  | [`import/order`]                                                                    |
-| [`prefer-function-over-method`]     | 🌟  | [`class-methods-use-this`][class-methods-use-this]                                  |
```

Comment:
[Non-actionable] Haha, I forgot about this file... good on you for remembering it!